fix: border-radius of comment-app

Rockpelt opened the pull request:
將留言區塊改為圓角方形，消除原本的白色角落。

修改前：
![image](https://user-images.githubusercontent.com/28527336/97156048-1f7f0380-17b1-11eb-9eb5-76f2dfe83b14.png)

修改後：
![image](https://user-images.githubusercontent.com/28527336/97156067-2574e480-17b1-11eb-9c88-9543a57c1615.png)

Rockpelt committed:
fix: border-radius of comment-app

modify the border-radius for containers in comment-app into 4px

titangene commented:
感謝發 PR，下面是 HackMD 留言區塊的 HTML：

```html
<div class="ui-comment-app small" id="hackmd-app">
  <div id="comment-app">
    <div class="open-comments">
      ...
    </div>
  </div>
</div>
```

HackMD 本身的 reset CSS 就有 `box-sizing` 的設定，所以不需要另外設定：

```css
* {
  box-sizing: border-box;
}
```

並且 CSS 選擇器 `.ui-comment-app *` 不需要使用 `*`，讓所以 `.ui-comment-app` 的後代都新增 `border-radius`，只需讓 `.open-comments` 新增即可：

```css
.open-comments {
  border-radius: 4px;
}
```

Rockpelt committed:
Fix border-radius of open-comments

Modify the border-radius as the maintainer suggests.

Rockpelt committed:
Fix boarder-radius of ui-comment-button-container

Fix the boarder-radius of ui-comment-button-container into 6px

Rockpelt committed:
Remove unintended line break

Rockpelt commented:
感謝修改建議！將相關段落照您的建議修改，並且將其移至程式碼處理 comment 部分的區段。

另外一併修改在 `ui-comment-button-container` 的 `border-radius`，使其與預設 stylesheets 當中的 `ui-comment-container`  一致。

修改前：
![image](https://user-images.githubusercontent.com/28527336/97964766-40b8a300-1df4-11eb-8b0d-cfcdd311354c.png)

修改後：
![image](https://user-images.githubusercontent.com/28527336/97964822-575efa00-1df4-11eb-99d5-7bfade82da65.png)

titangene commented:
感謝修正！

你指的 `.ui-comment-button-container` 主要是在解決 hover 會有白色角落嗎？

```css
.ui-comment-app .ui-comment-button-container.comment:not(.active):hover {
  z-index: 1050;
  background-color: white;
}
```

我會傾向將背景色設為透明，因為在原主題 hover 後變為白色背景也不會讓視覺有改變。

Rockpelt committed:
Revert till 38f78f

Revert to 38f78fef1fcd5f756e458b1b9f14aaa0d304d39d.

Rockpelt committed:
Set ui-comment-button-container into transparent background

Set the background-color of .ui-comment-button-container.comment into transparent, to fix the behavior happening upon hover and active.

Rockpelt committed:
Modify css selector for consistent code

Add the parent of open-comments in the selector.

Rockpelt commented:
感謝建議！

將 `.ui-comment-app .ui-comment-button-container.comment`  的背景設為透明，解決原本 active 和 hover 會有白色角落的狀況。

PS. 抱歉上個回應沒有敘述清楚。前則回應當中的圖片是 active 的狀況（我那時沒有注意到 hover 會有同樣的問題）。
